CSS/styles/style.css

```diff
@@ -0,0 +1,365 @@
+body {
+  font-size: 13px;
+  /*1rem = 16px*/
+}
+
+.container {
+  width: 1240px;
+  margin: 0 auto;
+}
+.clearfix::after {
+  content: '';
+  display: block;
+  clear: both;
+}
+a {
+  text-decoration: none;
+  color: #000000;
+}
+.page-header {
+  width: 100%;
+}
+.section-logo>a>image{
+  width: 315px;
+}
+.header-right {
+  float: right;
+  margin: auto 0;
+}
+.icon-contact {
+  background-image: url(../images/contact.png);
+  width: 40px;
+  height: 40px;
+  float: left;
+  margin-top: 15px;
+}
+.icon-contact,
+.header-right>p {
+  display: inline-block;
+}
+.header-right>p {
+  font-size: 14px;
+  font-style: italic;
+  font-weight: bold;
+  line-height: 70px;
+}
+.header-right>p>span {
+  font-size: 18px;
+  color: #4cc1fc;
+}
+.session-menu {
+  border: 1px solid grey;
+}
+.menu-list{
+  float: left;
+  display: -webkit-box;
+  line-height: 3.0;
+  width: 87%;
+}
+.menu-item{
+  display: block;
+  text-transform: uppercase;
+  padding: 22px;
+  font-size: 12px;
+  text-align: center;
+  border-right: 1px solid grey;
+}
+.no-border {
+  border: none;
+}
+.menu-item>a:hover {
+  color: #62c6f8;
+}
+.social-list{
+  float: right;
+  display: -webkit-box;
+}
+.social-item{
+  margin-top: 21px;
+}
+/*css main*/
+.banner {
+  width: 100%;
+  height: 160px;
+  background-image: url(../images/banner.png);
+  background-size: cover;
+}
+.banner>div.container>ul {
+  line-height: 160px;
+}
+.banner>div.container>ul>li {
+  text-transform: uppercase;
+  display: inline-block;
+  padding-left: 5px;
+  float: left;
+}
+.page-container {
+  margin-top: 75px;
+}
+.sidebar-left {
+  display: block;
+  float: left;
+  width: 300px;
+}
+.category-list {
+  padding-bottom: 45px;
+}
+.title-item {
+  width: 100%;
+  height: 50px;
+  display: inline-block;
+  background-color: #d9e4e9;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  padding: 10px;
+  line-height: 30px;
+  margin-bottom: 22px;
+}
+.title-item>li>a {
+  float: left;
+  font-weight: bold;
+}
+.title-item>li>span {
+  float: right;
+}
+.adults-list {
+  font-size: 12px;
+}
+.adults-list>li {
+  list-style-type: disc;
+  list-style-position: inside;
+  padding: 0 22px 22px;
+}
+.form-schedule {
+  padding: 20px;
+  box-sizing: border-box;
+  background-color: #d9e4e9;
+  text-align: center;
+}
+.form-schedule>h2 {
+  font-size: 18px;
+  text-transform: uppercase;
+  float: left;
+  margin-bottom: 15px;
+}
+.form-schedule>form>input {
+  width: 100%;
+  margin-bottom: 15px;
+  height: 35px;
+  box-sizing: border-box;
+}
+.form-schedule>form>button {
+  font-size: 18px;
+  text-transform: uppercase;
+  background-color: #4bbc56;
+  color: #ffffff;
+  width: 125px;
+  height: 45px;
+}
+.services-list {
+  width: calc(100% - 300px);
+  float: right;
+  display: inline-block;
```

Comment:
Should be `display: block`
No need in this case.

Reply:
Cảm ơn a, e sẽ chỉnh lại ạ!